tests/range_detection.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import numpy as np
+import pandas as pd
+
+from anomalydetection.detectors import RangeDetector, DiffRangeDetector, AnomalyDetectionPipeline
+
+
+@pytest.fixture
+def range_data():
+    normal_data = np.array([0, np.nan, 1, 0, 2, np.nan, 3.14, 4])
+    abnormal_data = np.array([-1, np.nan, 2, np.nan, 1, 0, 4, 10])
+    expected_anomalies = np.array([True, False, False, False, False, False, True, True])
+    assert len(expected_anomalies) == len(abnormal_data)
+    return normal_data, abnormal_data, expected_anomalies
+
+
+@pytest.fixture
+def range_data_series(range_data):
+    normal_data, abnormal_data, expected_anomalies = range_data
```

Comment:
range_data needs paranthesis for function to be called

Reply:
If we do then we get: TypeError: 'tuple' object is not callable